apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/GrunnlagsendringshendelseService.kt

```diff
@@ -333,6 +334,30 @@ class GrunnlagsendringshendelseService(
             }.map { it.first }
     }
 
+    fun opprettDoedshendelseForPerson(grunnlagsendringshendelse: Grunnlagsendringshendelse): OppgaveIntern? {
+        val eksisterendeHendelse =
+            grunnlagsendringshendelseDao.hentGrunnlagsendringshendelserMedStatuserISak(
+                grunnlagsendringshendelse.sakId,
+                listOf(GrunnlagsendringStatus.VENTER_PAA_JOBB, GrunnlagsendringStatus.SJEKKET_AV_JOBB),
+            ).firstOrNull {
+                it.gjelderPerson == grunnlagsendringshendelse.gjelderPerson && it.type == grunnlagsendringshendelse.type
+            }
+
+        // Vi bryr oss ikke om det samsvarer eller ikke, men underliggende tjenester krever det.
+        val samsvarMellomKildeOgGrunnlag = SamsvarMellomKildeOgGrunnlag.Doedsdatoforhold(null, null, false)
+        if (eksisterendeHendelse != null) {
+            logger.info(
+                "Det finens allede en grunnlagsendringshendelse ${grunnlagsendringshendelse.type} med id " +
+                    "${grunnlagsendringshendelse.id} fra før, forsøker å returnere relevant oppgave.",
+            )
+            return oppgaveService.hentOppgaverForSak(grunnlagsendringshendelse.sakId)
```

Comment:
Du kan hente oppgave for referanse og sjekke om den finnes i stedet for?

Reply:
Lurt 👍

Reply:
[703329e](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3927/commits/703329e94dde12c711eac56f47fe36e596dc9739)